_guides/responsive/control_layout.md

```diff
@@ -137,7 +121,7 @@ Depending on the screen width, one or the other will be fetched and rendered.
 </amp-img>
 {% endhighlight %}
 
-## Control element’s size and assets using srcset and sizes
+## Using srcset and sizes
```

Comment:
Need to mention that we support the `w` based srcsets in all browsers (Safari doesn't support it natively). Could we link to a good in-depth article on srcset?